test/instance.ts

```diff
@@ -438,9 +455,9 @@ describe('Instance', () => {
     it('should return any non-404 like errors', done => {
       const error = {code: 3};
 
-      instance.getMetadata = callback => {
-        callback(error);
-      };
+      sandbox
```

Comment:
It might be a good idea to add
```
afterEach(() => sandbox.restore());
```
In this `describe` block

Reply:
Done!!